modules/hitl2/src/translations/en.json

```diff
@@ -30,20 +31,26 @@
       "other": "Assigned to {name}"
     },
     "assign": "Escalation {id} successfully assigned",
-    "resolve": "Escalation {id} successfully resolved"
+    "resolve": "Escalation {id} successfully resolved",
+    "noPreviousAgent": "No prevous agent",
+    "you": "You",
+    "agent": "Agent: {agentName}"
   },
   "user": {
     "variables": {
       "heading": "Contact variables",
-      "empty": "No variables"
+      "empty": "No variables",
+      "variable": "Variable",
+      "value": "Value"
     }
   },
   "comments": {
     "heading": "Notes from agents",
     "empty": "No comments"
   },
   "conversation": {
-    "empty": "Select a pending case to engage conversation with your clients."
+    "empty": "Select a pending case to engage conversation with your clients.",
```

Comment:
We should recheck the vocabulary, escalations, handoffs, cases they're all thr same and it's quite confusing